types.go

```diff
@@ -97,7 +97,7 @@ var FileTypeFields = map[FileType][]string{
 		"comment",
 	},
 
-	ModelsFile: []string{
+	ModelsFile: {
```

Comment:
Does this type definition need to be changed to include the new `release_level` and `release_serial` fields?